package.json

```diff
@@ -96,7 +96,7 @@
     "test-ios": "./scripts/objc-test.sh test"
   },
   "workspaces": [
-    "packages/!(eslint-config-react-native-community)",
```

Comment:
Love this <3

Reply:
Less special cases!

It's always a little bit scary because *someone* at some point in time thought this was a good idea. I could not find any reason why this was originally done, though, so… move fast and break things, I suppose. 😁

Reply:
Oh, actually… I found the reason in https://github.com/facebook/react-native/commit/3967635632831b94293f0aff1daf5d2758e062fa:

> Note: We do not use `eslint-config-react-native-community` internally and it pulls in a lot of packages we don't need. It is part of the React Native repo because it is used in the RN app template but we may want to choose to move it out into a separate repo at some point.

Yeah… that's fine. We want these TypeScript dependencies installed now.